cli/cli.py

```diff
@@ -365,6 +358,11 @@ def amazonrequests(
     if not notification_handler.sound_enabled:
         log.info("Local sounds have been disabled.")
 
+    # don't run if `--test` and not `--use-atc-mode`
+    if test and not use_atc_mode:
+        log.error("Cannot run TURBO INITIATE in test mode; Use ATC mode for testing")
+        sys.exit(0)
+
```

Comment:
nitpicky, but shouldn't be this a separate PR?

Reply:
I think maybe just have a system pause here, rather than exit.